src/nav/AppContainer.js

```diff
@@ -66,6 +67,8 @@ class AppContainer extends PureComponent {
       actions.fetchEssentialInitialData();
       actions.fetchRestOfInitialData(pushToken);
       actions.fetchEvents();
+      actions.fetchUsersStatus(); // do a initial fetch
+      setInterval(() => actions.fetchUsersStatus(), config.activePingInterval);
```

Comment:
@vishwesh3 are you sure we need to request user status manually?
I am almost certain we get this via event presence updates.